Portfolio: Mobile headline

maccony202 opened the pull request:
I built a toolbar or header and also headline section using flexbox. and i stick to the design as much as possible using figma template.

Bondok6 reviewed (commented):
## Hi @maccony202 🙋‍♂️, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] Your `README` file should be descriptive - all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present 👌 
Please, update it 💯

- [ ] Also, I suggest updating the PR Summary description (should be more descriptive) 👍


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

Favourezeugwa commented on README.md:
```diff
@@ -1,2 +1,46 @@
+
+
+👤 **Author1**
+- Collins Nana Antwi Baffour-Gyawu
```
- I think it will be better if you replaced `**Author1**` with `Collins Nana Antwi Baffour-Gyawu` as you are the author of this project. 🤗

Favourezeugwa commented on index.html:
```diff
@@ -0,0 +1,41 @@
+                <li><a href="#"><img src="images/Vector.png" alt=""></a></li>
+            </ul>
+        </div>
+    </header>
```
- I think this entire section should NOT be in a `<header>` tag, because typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. 💪🏽 

I will point out to you where you should add the `<header>`. But you'd need to follow through with my suggestions to achieve this. 🌸

Favourezeugwa commented on index.html:
```diff
@@ -0,0 +1,41 @@
+                <img class="logo" src="images/John Doe (1).png" alt="">
+                <img class="nav-icon" src="images/Union.png" alt="">
+            </nav>
+        </div>
```
- I think your `<div class="nav-container">` should be replaced with the `<header>` tag, as this defines your page title and logo 🍷 

-  On line 14, I can see that you have your Logo as an image. I think it is supposed to be a `text`  and should be in a link tag `<a>` for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page). 🌸 
- Likewise, I think it is preferrable for your hamburger menu to be in a `<nav>` tag, and not the entire Logo and menu for semantics reason and for a better user experience on mobile devices. 😉 

You can kindly re-factor your code to look something like this instead 👇🏽 

```
         <header class="nav-container">
            <a href="#" class="logo">John Doe</a>
            <nav>
                <img class="nav-icon" src="images/Union.png" alt="">
            </nav>
        </header>
```

What do you think about this?? I think you can also replace John Doe with your name 👍🏽

Favourezeugwa commented on index.html:
```diff
@@ -0,0 +1,41 @@
+                <img class="nav-icon" src="images/Union.png" alt="">
+            </nav>
+        </div>
+        <h1>
+            Hey There. I’m Nana
+        </h1>
+        <h2>
+            I’am a Software Developer
+        </h2>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        </p>
+        <div class="social-icons">
+            <ul>
+                <li><a href="#"><img src="images/Vector4.png" alt=""></a></li>
+                <li><a href="#"><img src="images/Group.png" alt=""></a></li>
+                <li><a href="#"><img src="images/Vector2.png" alt=""></a></li>
+                <li><a href="#"><img src="images/Vector1.png" alt=""></a></li>
+                <li><a href="#"><img src="images/Vector.png" alt=""></a></li>
+            </ul>
+        </div>
```
- I think it will be better if you wrap this entire section in a `<section class=" introduction">` tag, as this section contains an introduction about you, your bio, what you do, and social media handles. 

- I believe this part needs to have a background image as well. You can simply add a background image on your CSS file after wrapping up your entire introduction in a section tag. 🍷 

| Figma Template Design  |Your Design |
| :---:   | :---: | 
|![desktopbg](https://user-images.githubusercontent.com/99246359/198818755-7d98c505-9a16-4964-8ec3-571d5525ebba.PNG)  |![hehas](https://user-images.githubusercontent.com/99246359/198818773-8862d73c-fa92-4f25-9588-a069bdb09224.PNG)  |  


- I think you have the image saved in your images folder as `illustration-header-mobile.png` 🌸 

So you can do this 👇🏽 

```
.introduction {
  other styles: here;
  background: url("images/illustration-header-mobile.png");
}

```

Favourezeugwa reviewed (changes requested):
## STATUS: REQUIRES CHANGES ♻️  🚦 

Hi @maccony202 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![woman so close](https://user-images.githubusercontent.com/99246359/198819353-6951cd88-5a45-4ddb-aadf-20a1c35125e0.gif)

## Highlights 💯 
- No linter errors ✔️ 
- Professional Readme file ✔️ 
- Great commit messages ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Lameck1 reviewed (commented):
Hello @maccony202 👋,

_This is an additional review as a result of the second opinion request._

Your main complaint with the last review is that the requested changes are not clear to you. In this case, we always advise that you reach out to the code reviewer to point out which specific concern is not clear to you.

I hope you realize that you can write a reply just below the reviewer's comment. Please use that feature and @Favourezeugwa will get back to you.

Keep rocking 🚀!

Kossi-stack commented on index.html:
```diff
@@ -0,0 +1,46 @@
+<body>
+    <header>
+        <div class="nav-container">
+            <nav>
```
- [ ] Kindly wrap the menu into a nav tag
- And the Logo which is your name in a tag separately don't put both in the nav tag is wrong

maccony202 replied:
boss i dont get it can please explain further because i only have 1 review slot

Kossi-stack commented on README.md:
```diff
@@ -1,2 +1,45 @@
+- Linters
+
+
+## Authors
```
- [ ] As stated by the previous reviewer he said replace so kindly delete line 14.

Lameck1 replied:
@Kossi-stack, line 14 is the heading for the `Authors` section, it is not supposed to be deleted. The previous reviewer only suggested that @maccony202 should replace the generic name `Author1` with a proper name. @maccony202, I am sorry this has made your README look not organized. Please consider reverting this change.

Kossi-stack reviewed (changes requested):
## Hi @,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

![68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966](https://user-images.githubusercontent.com/81444338/170121142-a9f93bb7-38bb-45cc-8327-aab2501ef90a.gif)

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

 - [OPTIONAL] Kindly edit the PR's title to something more descriptive. it can be "Portfolio Setup PR ".
 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

ElsonOtake commented on README.md:
```diff
@@ -1,2 +1,41 @@
+- HTML & CSS
+- Linters
+
+👤 Collins Nana Antwi Baffour-Gyawu
```
- [OPTIONAL] I think it's better to use `**Collins Nana Antwi Baffour-Gyawu**` as the author. Will create a more professional look for your README. For more information [click here](https://www.markdownguide.org/cheat-sheet/).

ElsonOtake commented on index.html:
```diff
@@ -0,0 +1,46 @@
+</head>
+<body>
+    <header>
+        <menu class="nav-container">
+            <h1><a href="#">Nana Antwi</a></h1>
+            <nav>
+                <img class="nav-icon" src="images/Union.png" alt="">
+            </nav>
+        </menu>
```
- [ ] Kindly use the suggested structure provided by the previous reviewer.
```
         <header class="nav-container">
            <a href="#" class="logo">John Doe</a>
            <nav>
                <img class="nav-icon" src="images/Union.png" alt="">
            </nav>
        </header>
```
Use the appropriate tags for each element. Follow the [HTML best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

maccony202 replied:
Sir please  why is everyone suggesting different thing. Is like I make changes base on reviewer’s suggestions and another reviewer will come and make different suggestions

Lameck1 replied:
@maccony202, it is not best practice to have two `<h1>` tags on the same HTML page. It is not good for SEO. Please refactor `<h1><a href="#">Nana Antwi</a></h1>` to `<a href="#">Nana Antwi</a>`.

ElsonOtake commented on index.html:
```diff
@@ -0,0 +1,46 @@
+                <li><a href="#"><img src="images/Vector2.png" alt=""></a></li>
+                <li><a href="#"><img src="images/Vector1.png" alt=""></a></li>
+                <li><a href="#"><img src="images/Vector.png" alt=""></a></li>
+                <div class="back">
+                    <img src="images/illustration-header-mobile.png" alt="">
+                </div>
```
- [ ] Please use the image as a background defined on your CSS file as suggested by the previous reviewer. 
```
.introduction {
  other styles: here;
  background: url("images/illustration-header-mobile.png");
}
```
For more information [click here](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image).

Lameck1 replied:
- [ ] Just like it was requested by the first reviewer, it is important that you add this image as a background image, not directly into the HTML. Please consider this.

ElsonOtake reviewed (changes requested):
STATUS: CHANGES REQUIRED 🟡 

Hi @maccony202, 

Great work on making the changes requested by a previous reviewer 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🥲 🥲 

![woohoo-youre-almost](https://user-images.githubusercontent.com/77038610/154921145-1f7bb8f6-c843-4ba5-a534-6b840c600558.jpeg)

 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. 👍 👍 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Lameck1 commented on index.html:
```diff
@@ -0,0 +1,46 @@
+<body>
+    <header>
+        <menu class="nav-container">
+            <h1><a href="#">Nana Antwi</a></h1>
```
- [ ] @maccony202, it is not best practice to have two `<h1>` tags on the same HTML page. It is not good for SEO. Please refactor `<h1><a href="#">Nana Antwi</a></h1>` to `<a href="#">Nana Antwi</a>`.

Lameck1 reviewed (commented):
Hello @maccony202  👋,

_This is an additional review as a result of the second opinion request._

I can attest that you have worked on the changes suggested by @Kossi-stack. @ElsonOtake you were supposed to be consistent with @Kossi-stack's review, not all previous reviews.
 
As this was the only reason that your project was not approved it will be approved as soon as possible.

Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away.

Note: Please consider the changes suggested by @ElsonOtake nonetheless, they will make you project look professional.

Keep rocking 🚀!

ha-manel reviewed (changes requested):
# STATUS: REQUIRED CHANGES ♻️

## Hi @maccony202 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  

<img src="https://media3.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif"/>

## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 

## Required Changes:
- [ ] Kindly update the pull request's description. It should have a descriptive summary of the changes made in this branch, ex: Added navbar, Added headline, Used mobile first approach ... etc.
- [ ] Kindly update the pull request's title, it should be descriptive (ex: Portfolio: Mobile headline) 👍 
- [x] Kindly remove the white empty space on the right of the page 👍 

![image](https://user-images.githubusercontent.com/50721479/199223346-51aa87b7-463f-441d-9623-815a917867a0.png)


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

maccony202 commented:
> # STATUS: REQUIRED CHANGES ♻️
> ## Hi @maccony202 👋
> Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍
> 
> ![](https://camo.githubusercontent.com/9e159f4c169d0a1e414deb7f91532bcacdf831860b1c3b10e6b629a2ee421975/68747470733a2f2f6d65646961332e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966) [ ![68747470733a2f2f6d65646961332e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966](https://camo.githubusercontent.com/9e159f4c169d0a1e414deb7f91532bcacdf831860b1c3b10e6b629a2ee421975/68747470733a2f2f6d65646961332e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966) ](https://camo.githubusercontent.com/9e159f4c169d0a1e414deb7f91532bcacdf831860b1c3b10e6b629a2ee421975/68747470733a2f2f6d65646961332e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966) [ ](https://camo.githubusercontent.com/9e159f4c169d0a1e414deb7f91532bcacdf831860b1c3b10e6b629a2ee421975/68747470733a2f2f6d65646961332e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights:
> * All tests passed ✔️
> * Meaningful commit messages ✔️
> 
> ## Required Changes:
> * [ ]  Kindly update the pull request's description. It should have a descriptive summary of the changes made in this branch, ex: Added navbar, Added headline, Used mobile first approach ... etc.
> * [ ]  Kindly update the pull request's title, it should be descriptive (ex: Portfolio: Mobile headline) 👍
> * [ ]  Kindly remove the white empty space on the right of the page 👍
> 
> ![image](https://user-images.githubusercontent.com/50721479/199223346-51aa87b7-463f-441d-9623-815a917867a0.png)
> 
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

please sir there is no empty space on my end. it is responsive

nicupop729 reviewed (commented):
Hello @maccony202  👋,

_This is an additional review as a result of the second opinion request._

Kudos for the good progress you have done until now in this milestone :clap:

About the requirements mentioned by the previous reviewer, I need to comment on the followings:
I cannot see that white empty space on the right of the page, so your progress should not be stopped because of this aspect.

Regarding the other aspects mentioned by the reviewer, they are all valid and you need to improve the names of the PRs, branches and the description of the PRs.

The names need to be meaningful and descriptive.
Examples of names for this branch that you could have used:
"create_the_nav_and_headline", or just as simple as "nav_and_headline".
Example of name for this PR:
"Build navigation and headline for mobile view"
Example of a good description of the PR:
"
## This branch contains the following improvements: 

I built the toolbar (or header).
Also the headline section (right after the header).

In order to lay out the elements on the page I used Flexbox in all 2 sections.
I stick to the design as much as possible (e.g. font, colours, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
"

I trust that from now on you will use better names for your branches and PRs and also a better description.
Please note that bad names and missing descriptions are reasons for the invalidation of the PR and the code reviewers in the next milestones will definitely invalidate it if names and descriptions are not descriptive and meaningful.

Please find more details about how to document your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

### As I recognise your effort and your progress, I would say your PR is ready to be approved this time.

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!
